core/src/test/java/org/kiwix/kiwixmobile/core/dao/HistoryDaoTestUtil.kt

```diff
@@ -0,0 +1,19 @@
+package org.kiwix.kiwixmobile.core.dao
+
+import io.mockk.every
+import io.mockk.mockk
+import org.kiwix.kiwixmobile.core.history.HistoryListItem
+
+fun mockkHistoryItemWithTitle(historyTitle: String): HistoryListItem.HistoryItem {
```

Comment:
Why a separate file for this? Is it a convention to do so?

Reply:
My thinking was that multiple utility functions will bloat up the test class and make it less readable. Did not know at the time whether I'd create more utility functions but see now that by utilizing the data class, this file will be removed.